WordPress/src/main/java/org/wordpress/android/util/WPUrlUtils.java

```diff
@@ -0,0 +1,45 @@
+package org.wordpress.android.util;
+
+import java.net.URI;
+import java.net.URL;
+
+public class WPUrlUtils {
```

Comment:
It should, because we're using this code in a multithread env. The app has background calls and we need to be sure we're not getting concurrency issues.